homeassistant/components/homekit/accessories.py

```diff
@@ -65,10 +65,10 @@ def _set_services(self):
 
     def run(self):
         """Method called by accessory after driver is started."""
-        state = self._hass.states.get(self._entity_id)
+        state = self.hass.states.get(self.entity_id)
```

Comment:
Maybe pass `hass` and `entity_id` into the `HomeAccesory` constructor, then you can remove those assignments from all child classes & you dont need the no-member pylint directive